pages/k8s/install-manual.md

```diff
@@ -0,0 +1,244 @@
+| 1.12.x         | [canonical-kubernetes-296](https://api.jujucharms.com/charmstore/v5/~containers/bundle/canonical-kubernetes-296/archive/bundle.yaml?channel=stable) |
+| 1.11.x         | [canonical-kubernetes-217](https://api.jujucharms.com/charmstore/v5/~containers/bundle/canonical-kubernetes-217/archive/bundle.yaml?channel=stable) |
+| 1.10.5+         | [canonical-kubernetes-211](https://api.jujucharms.com/charmstore/v5/~containers/bundle/canonical-kubernetes-211/archive/bundle.yaml?channel=stable)  |
+| 1.10.x         | [canonical-kubernetes-188](https://api.jujucharms.com/charmstore/v5/~containers/bundle/canonical-kubernetes-188/archive/bundle.yaml?channel=stable) |
+| 1.9.3+         | [canonical-kubernetes-179](https://api.jujucharms.com/charmstore/v5/~containers/bundle/canonical-kubernetes-179/archive/bundle.yaml?channel=stable) |
+| 1.9.x        | [canonical-kubernetes-150](https://api.jujucharms.com/charmstore/v5/~containers/bundle/canonical-kubernetes-150/archive/bundle.yaml?channel=stable) |
+| 1.8.x | [canonical-kubernetes-132](https://api.jujucharms.com/charmstore/v5/~containers/bundle/canonical-kubernetes-132/archive/bundle.yaml?channel=stable) |
+| 1.7.x | [canonical-kubernetes-101](https://api.jujucharms.com/charmstore/v5/~containers/bundle/canonical-kubernetes-101/archive/bundle.yaml?channel=stable) |
+| 1.6.x | [canonical-kubernetes-38](https://api.jujucharms.com/charmstore/v5/~containers/bundle/canonical-kubernetes-38/archive/bundle.yaml?channel=stable) |
+
+<div class="p-notification--caution">
+  <p markdown="1" class="p-notification__response">
+    <span class="p-notification__status">Note:</span>
+Only the latest three versions of CDK are supported at any time.
+  </p>
+</div>
+
+### Additional configuration
+
+To allow Kubernetes to access resources and functionality of the underlying
+cloud it is deployed on, additional integrator charms are available. When
+installing with **conjure-up**, these charms are automatically added to the
+deployment and configured accordingly. This table explains which charms are
+used:
+
+| AWS        | aws-integrator       | juju deploy cs:~containers/aws-integrator       | https://jujucharms.com/u/containers/aws-integrator/       |
+|------------|----------------------|-------------------------------------------------|-----------------------------------------------------------|
+| Azure      | azure-integrator     | juju deploy cs:~containers/azure-integrator     | https://jujucharms.com/u/containers/azure-integrator/     |
+| CloudSigma | openstack-integrator | juju deploy cs:~containers/openstack-integrator | https://jujucharms.com/u/containers/openstack-integrator/ |
+| Google     | gcp-integrator       | juju deploy cs:~containers/gcp-integrator       | https://jujucharms.com/u/containers/gcp-integrator/       |
+| Joyent     |                      |                                                 |                                                           |
```

Comment:
@tvansteenburgh Sorry, it is difficult to see here because its a table. Basically we say we 'support' various clouds, including Joyent, Cloudsigma, Oracle. There are no integrator charms for these though. Are they not needed? I'm guessing that rackspace can use the OpenStack one, but that is a guess.